README.md

```diff
@@ -102,21 +102,21 @@ As a JavaScript React component, the project is designed to be used in two diffe
 
 ## Flags
 
-The following flags are available in the latest release to toggle experimental features:
+The following flags are available to toggle experimental features:
 
-- `newgraph` - Improved graphing algorithm (default `false`).
+- `newgraph` - From release v3.3.2. Improved graphing algorithm (default `false`).
```

Comment:
Yeah I'd say `3.4.0`, though I doubt it's necessary to specify as it'll be in the release notes, and this will only be visible on branches that contain this flag, no?

Reply:
Yeah I was thinking on the same lines but this was a suggestion from @stichbury earlier in this thread and it seems a good thing to keep - I've updated this to read `3.4.0`.